topo_processor/metadata/metadata_validators/tests/metadata_validator_stac_local_schema_test.py

```diff
@@ -0,0 +1,125 @@
+import os
+from datetime import datetime
+
+import pytest
+
+import topo_processor.stac as stac
+from topo_processor.metadata.metadata_validators.metadata_validator_stac import MetadataValidatorStac
+from topo_processor.stac.validate_report import ValidateReport
+
+
+@pytest.mark.skip(reason="use this test for local schema testing - multiple errors in report")
```

Comment:
It's good to have an example. I feel they all look the same? I think one example would be enough?

I get your point to have a specific test file for local tests but `[python_file_to_test]_test.py` is a naming convention for unit tests. As you have put this test in another test file called `metadata_validator_stac_local_schema_test.py`, we lost this naming convention.